bgpd/bgp_evpn_vty.c

```diff
@@ -3462,7 +3462,7 @@ DEFUN(show_bgp_l2vpn_evpn_vni,
 
 	uj = use_json(argc, argv);
 
-	bgp_def = bgp_get_default();
+	bgp_def = bgp_get_evpn();
```

Comment:
rename the variable otherwise it will just be a source of confusion.

Reply:
All variables are renamed in the last patch of the series, to keep individual commits shorter and easier to review. I can apply back this patch to individual commits where it makes sense if you prefer.

Reply:
ack